middleware/src/middleware.go

```diff
@@ -264,7 +295,10 @@ func (middleware *Middleware) BackupHSMSecret() rpcmessages.ErrorResponse {
 			Code:    errorCode,
 		}
 	}
-
+	err = middleware.redisClient.SetString(redis.BaseSetupDone, "1")
```

Comment:
Is `BackupHSMSecret()` the right place for this? I don't think this is even called in the current setup Wizard. Should probably have something explicit that is called by the App once the setup can be finalized.

Reply:
Ah, this is a bad mistake. I need something that detects when the initial setup is done in the frontend. Should I move it to BackupSysconfig instead? @Stadicus do you have an idea what could act as a trigger to mark the base as initialized?

Reply:
We've discussed something similar this morning with @Stadicus @Tomasvrba. We don't want to restart bitcoind multiple times when applying the changes made in the setup wizard, because it takes time. One idea would be to have a RPC that finalizes the setup process, sets the redis key(s) and restarts bitcoind once.

Reply:
As per offline discussion I'm moving it to BackupSysconfig.